samples/hello-oboe/src/main/cpp/PlayAudioEngine.h

```diff
@@ -50,15 +50,30 @@ class PlayAudioEngine : public AudioEngine<SoundGenerator, LatencyTuningCallback
 
     void setBufferSizeInBursts(int32_t numBursts);
 
-    // Used to display latency in the app
-
+    /**
```

Comment:
I'd prefer this comment block to be kept with the implementation - I often refer developers to it and it's easier to understand when both comment and code are together.